main/webapp/modules/core/scripts/views/data-table/cell-ui.js

```diff
@@ -99,25 +99,35 @@ DataTableCellUI.prototype._render = function() {
     } else {
       var arr = cell.v.split(" ");
       var spanArr = [];
-      for (var i=0; i<arr.length; i++){
+      for (var i=0; i<arr.length; i++) {
         if (URL.looksLikeUrl(arr[i])) {
           if (spanArr.length != 0) {
             var span = document.createElement('span');
             span.textContent = spanArr.join(" ");
             divContent.appendChild(span).appendChild(document.createTextNode('\u00A0'));
             spanArr = [];
           }
+          var [realURL, extra] = (() => {
+            var parts = str.split('\n');
```

Comment:
You may want to adjust this from your previous update :)
```suggested
var parts = arr[i].split('\n');
```

Reply:
Done